pytorch_lightning/trainer/supporters.py

```diff
@@ -150,7 +263,7 @@ def add(self, predictions):
     def to_disk(self) -> None:
```

Comment:
```suggestion
    def to_file(self) -> None:
```
since user may ask if there another func for the cloud?

Reply:
It would break BC.